canonical-web-and-design/jaas-dashboard
# Add machines and units panels to model details pages

## Done

Add machines and units panels to model details pages

Note; the tables in these panels will not populate as we need to refactor modelStatusData before we can do that. @hatched and I have discussed this in Mattermost. The units and machines panels will not be exposed on production.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click a model
- Click the 'Machines' filter and click a machine - you should see a panel open with relevant data
- Click the 'Units' filter and click a unit - you should see a panel open with the relevant data

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1369,
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1370,
